lib/quicly.c

```diff
@@ -284,9 +301,28 @@ struct st_quicly_conn_t {
          */
         int64_t send_ack_at;
         /**
-         *
+         * congestion control
          */
         quicly_cc_t cc;
+        /**
+         * pending NEW_CONNECTION_ID
+         */
+        struct {
+            /**
+             * maximum number of path ID issued so far
+             */
+            uint32_t max_path_id;
+            /**
+             * list of struct st_quicly_new_cid_t
+             */
+            quicly_linklist_t head;
+        } new_cid;
```

Comment:
@kazuho Should these (`new_cid` and `retire_cid` structures here) under `pending` instead of `egress`?

Reply:
Okay I think I got the idea. Let me work on this.

In the meantime, do you have any opinion on where to place `struct { ... } retire_cid`?

Reply:
How about making it a member of `egress`? I think we might want to merge `penging` into `egress`.